src/Builder/DatabaseBuilder.php

```diff
@@ -0,0 +1,239 @@
+    {
+        $this->payload['is_inline'] = true;
+
+        return $this;
+    }
+
+    /**
+     * Sets the icon for the database creation.
+     *
+     * @param  string  $icon
+     * @return DatabaseBuilder
+     */
+    public function iconEmoji(string $icon): DatabaseBuilder
+    {
+        $this->payload['icon'] = [
+            'type' => 'emoji',
+            'emoji' => $icon,
+        ];
+
+        return $this;
+    }
+
+    /**
+     * Sets the icon for the database creation.
+     *
+     * @param  string  $url
+     * @return DatabaseBuilder
+     */
+    public function iconExternal(string $url): DatabaseBuilder
+    {
+        $this->payload['icon'] = [
+            'type' => 'external',
+            'external' => [
+                'url' => $url,
+            ],
+        ];
+
+        return $this;
+    }
+
+    /**
+     * Sets the cover for the database creation.
+     *
+     * @param  string  $url
+     * @return DatabaseBuilder
+     */
+    public function coverExternal(string $url): DatabaseBuilder
+    {
+        $this->payload['cover'] = [
+            'type' => 'external',
+            'external' => [
+                'url' => $url,
+            ],
+        ];
+
+        return $this;
+    }
+
+    /**
+     * Adds the property `title` database creation.
+     *
+     * @param  string  $name
+     * @return DatabaseBuilder
+     */
+    public function addTitle(string $name = 'Name')
+    {
+        $this->add(PropertyBuilder::title($name));
+
+        return $this;
+    }
+
+    /**
+     * Adds one or multiple properties to the database creation.
+     *
+     * @param  PropertyBuilder|Collection|DatabaseSchemeBuilder  $properties
+     * @return DatabaseBuilder
+     */
+    public function add(PropertyBuilder|Collection|DatabaseSchemeBuilder $properties): DatabaseBuilder
+    {
+        if ($properties instanceof PropertyBuilder) {
+            $properties = collect([$properties]);
+        }
+
+        if ($properties instanceof DatabaseSchemeBuilder) {
+            $properties = $properties->getProperties();
+        }
+
+        $properties->each(function (PropertyBuilder $property) {
+            $this->payload['properties'][$property->getName()] = $property->payload();
+        });
+
+        return $this;
+    }
+
+    /**
+     * Adds multiple properties to the database creation, similar to a Laravel migration.
+     *
+     * @param  callable  $callback
+     * @return DatabaseBuilder
+     */
+    public function scheme(callable $callback): DatabaseBuilder
+    {
+        $builder = new DatabaseSchemeBuilder();
+        $callback($builder);
+
+        return $this->add($builder);
+    }
+
+    /**
+     * Adds a raw property to the database creation.
+     *
+     * @param  string  $title
+     * @param  string  $propertyType
+     * @param  array|null  $content
+     * @return DatabaseBuilder
+     */
+    public function addRaw(string $title, string $propertyType, array $content = null): DatabaseBuilder
```

Comment:
I think it should be `?array $content = null` right? 🤔

Reply:
yep!